### PR TITLE
SEDs with some styling updates

### PR DESCRIPTION
**Pull request by ailZhou**
### Description

SEDs but looking like it got some sleep.

I don't know what SEDs look like before this but I bet you it's never looked better.

A mural showing most of the pages that got updated. Also helps get a general sense of what has been removed/changed.
https://app.mural.co/t/coformaco8066/m/coformaco8066/1734714843135/15edbc9fd61c7048b604f32e930be23fda264af0?sender=u66d9aeebc714573cd17a9651

Things I may change in the future:
- Adding a better breadcrumb menu for admin
- The loading gear's color
- Confirming that the footer can match the one for the other sites
- Mobile friendliness ???

### Related ticket(s)

---
### How to test

Deploy Link: https://d1u531858aonal.cloudfront.net/

Sign into SEDs admin and look around. Look for any pages that may have gotten unreadable from the styling updates. 

### Notes

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/additional-scss/_header.scss [hunk 1]
```diff
@@ -40,3 +42,7 @@
     @extend .font-alt-lg;
   }
 }
+
+.usa-banner__inner {
+  margin: 0;
+}
```
Comment (benmartin-coforma) [inline services/ui-src/src/additional-scss/_header.scss:48]: I don't know about this one. I checked cms.gov, hhs.gov, irs.gov, va.gov, ed.gov... They all have the same banner, with the same wonky margin on the left side.

File: services/ui-src/src/additional-scss/_buttons.scss [hunk 2]
```diff
@@ -5,7 +5,8 @@
 .usa-button {
   background: $lochmara;
   width: 140px;
-  box-shadow: 5px 10px 8px #888888;
+  border: none;
+  margin: 0;
 
   &.hollow {
     background-color: $white;
```
Comment (benmartin-coforma) [inline services/ui-src/src/additional-scss/_buttons.scss:12]: Honestly we can probably delete all of our `usa-button` styles. We only have one .usa-button on the whole site, and it's the weird non-button in the Quarterly Forms table. If you don't do that in this PR, I'll do it soon in one of mine.

File: services/ui-src/src/animations/transitions.scss
Comment (benmartin-coforma) [inline services/ui-src/src/animations/transitions.scss:1]: Yes... ha ha ha... YES!

File: services/ui-src/src/components/FormPage/FormPage.scss [hunk 3]
```diff
@@ -11,3 +11,17 @@
 .action-button {
   margin: 0.6rem 0rem 2rem 5rem;
 }
+
+.formfooter{
+  .grid-container{
+    max-width: none;;
```
Comment (benmartin-coforma) [inline services/ui-src/src/components/FormPage/FormPage.scss:17]: Request: Would you mind removing the double `;;` here, and adding a newline at the end of the file?

File: services/ui-src/src/components/Login/Login.scss [hunk 4]
```diff
@@ -7,9 +7,14 @@
...
-    margin-left: 39%;
+.login-wrapper {
+  form {
+    label {
+      width: 100%;
+      text-align: left;
+    }
+    .form-control {
+      width: 100%;
+    }
```
Comment (benmartin-coforma) [inline services/ui-src/src/components/Login/Login.scss:18]: Thought: At 100% width, the text boxes are 8 pixels wider than the buttons above and below. Should we compensate here? `width: calc(100% - 0.5rem);`

Or is there a better way?

(Or does 8 pixels not matter compared to the 2 inch offset we had before?)

**Review by benmartin-coforma — COMMENTED** [review]
Wow, SEDS, you look great. Did you do something with your hair?

File: services/ui-src/src/additional-scss/_header.scss (hunk 1 above)
Comment (ailZhou) [reply services/ui-src/src/additional-scss/_header.scss:48]: i will revert it begrudgingly 😞

File: services/ui-src/src/components/Login/Login.scss (hunk 4 above)
Comment (ailZhou) [reply services/ui-src/src/components/Login/Login.scss:18]: checked with Ben on slack and this is good

**Review by benmartin-coforma — APPROVED** [review]
This is looking REALLY good, a 500% improvement visually.

I poked around the site some more, and the only thing that still stands out to me is the alignment of the button on the Signup page:

image

File: services/ui-src/src/components/GenerateForms/GenerateForms.jsx [hunk 5]
```diff
@@ -73,28 +73,28 @@ const GenerateForms = () => {
...
       ) : null}
-
-      <p className="margin-bottom-3">
+      <h1 className="page-header">Generate Quarterly Forms</h1>
+      <p className="margin-bottom-4">
         Create new forms for each state by filling out the form below. Please
         select the year and quarter you wish to create form template from.
       </p>
-      <p>Select the Year</p>
+      <label>Select the Year</label>
```
[the remark this answers is not in the document]
Comment (ailZhou) [reply services/ui-src/src/components/GenerateForms/GenerateForms.jsx]: both of these are fixed now!

**Review by benmartin-coforma — APPROVED** [review]
Approval number 3! I didn't log in to see the final versions myself, but I don't see any issues in the code and your screenshots look amazing.

**Review by rocio-desantiago — APPROVED** [review]
Poked around and actions and visuals working as expected

**Pull request merged**